app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
How about renaming `ManageIQ::Providers::AutomationManager::Authentication` to `ManageIQ::Providers::AutomationManager::Credential` ?